drivers/dac/dac_ad56xx.c

```diff
@@ -34,7 +34,7 @@ struct ad56xx_config {
 	const struct gpio_dt_spec gpio_reset;
 	uint8_t resolution;
 
-	uint8_t *channel_addresses;
+	const uint8_t *channel_addresses;
```

Comment:
This change should get squashed into the previous commit

Reply:
done